File: CleanAddisBackOfficeApi/src/main/java/com/iyoa/cleanaddis/json/CommentJson.java
package com.iyoa.cleanaddis.json;
import java.sql.Date;
import java.util.UUID;

import javax.persistence.Column;
import javax.persistence.FetchType;
import javax.persistence.GeneratedValue;
import javax.persistence.Id;
import javax.persistence.JoinColumn;
import javax.persistence.OneToOne;

import org.hibernate.annotations.GenericGenerator;

import com.fasterxml.jackson.annotation.JsonIgnore;
import com.fasterxml.jackson.annotation.JsonProperty;
import com.iyoa.cleanaddis.model.Post;
import com.iyoa.cleanaddis.model.User; 

import lombok.Data;
@Data
public class CommentJson {	 
	
	@JsonProperty("text")
	private String text;
	
	@JsonProperty("postUUID")
	private UUID postUUID;
	
	@JsonProperty("status")
	private String status;
	
	@JsonProperty("noLike")
	private int noLike;
	
	@JsonProperty("username")
	private String username;
	
	@JsonProperty("commentDate")
	private Date commentDate;
}

File: app/src/main/java/com/iyoa/cleanaddis/data/posting/CommentJSON.kt
package com.iyoa.cleanaddis.data.posting

import androidx.room.PrimaryKey
import com.squareup.moshi.Json
import java.util.*

data class CommentJSON(@Json(name="uuid") val  uuid: String,
                       @Json(name="text") val text:String,
                       @Json(name="post") val  postUuid: String,
                       @Json(name="status") val status: String,
                       @Json(name="noLike") val noLike: Int,
                       @Json(name="user") val commenterUuid:String,
                       @Json(name="commentDate") val date: String
){}